docs/build_developer_images.md

```diff
@@ -0,0 +1,64 @@
+# OpenShift Router Images
+
+There are situations in which you only need to recreate 
```

Comment:
This guide describes how to build specific OpenShift images, and outlines how to make use of them.